src/models/migrations/004_add_pools_table.sql

```diff
@@ -0,0 +1,12 @@
+-- +goose Up
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
If its one-many relation might as well keep user reference inside the ticket and skip one table